examples/gadgets/trace_dns/main.go

```diff
@@ -58,7 +59,8 @@ func do() error {
 
 	// Create the local manager operator
 	localManagerOp := localmanager.LocalManagerOperator
-	localManagerParams := localManagerOp.GlobalParamDescs().ToParams()
+	localManagerParams := apihelpers.ToParamDescs(localManagerOp.GlobalParams()).ToParams()
```

Comment:
I'm not sure if we want to introduce this change for API users right now, specially because we need to understand what to do with ParamDesc before (https://github.com/inspektor-gadget/inspektor-gadget/pull/4935#discussion_r2379992422), so I'd avoid to introduce breaking changes for now until we figure out the full clean up.